<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinishedListWidgetItemEdit</class>
 <widget class="QWidget" name="FinishedListWidgetItemEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="waitingWidgetHeaderEditFrame" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FAFBFC;
border-radius: 4px;
color: black;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource/res.qrc">
          <normaloff>:/style/image/back.png</normaloff>:/style/image/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Абортация</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#cancelButton
{
background-color: #e5ebf1;
color: #55677D;
border-radius: 4px;
padding: 7px 16px 8px;
}

QPushButton#cancelButton:hover
{
	text-decoration: underline;
}

QPushButton#cancelButton:pressed
{

}</string>
           </property>
           <property name="text">
            <string>Отменить все изменения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backToWaitingButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
background-color: #5181b8;
color: white;
padding: 7px 16px 8px;
border-radius: 4px;
}

QPushButton:hover
{
	background-color: #658FC0;
}
QPushButton:pressed
{
	background-color: #4a76a8;
}</string>
           </property>
           <property name="text">
            <string>Вернуть в Ожидание</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 4px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane 
{ 
    border-top: 1px solid #dce1e6;
	position: absolute;
}

QTabWidget:tab-bar 
{
	left: 10px;
}

QTabBar::tab
{
	color: #626d7a;
	padding: 18px 6px 20px;
	margin: 0 4px;
	border-bottom: 2px solid white;
}

QTabBar::tab:hover:selected
{
	border-bottom: 2px solid #5181b8;
	color: black;
}

QTabBar::tab:selected
{
	border-bottom: 2px solid #5181b8;
	color: black;
}

 QTabBar::tab:hover
{
	border-bottom: 2px solid #C5D0DB;
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Группы без ошибок</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QWidget" name="noErrorGroupList" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2"/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Группы с ошибкой</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWidget" name="errorGroupList" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3"/>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource/res.qrc"/>
  <include location="resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
